<ui version="4.0" >
 <class>AccountManageClass</class>
 <widget class="QWidget" name="AccountManageClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AccountManage</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="trwAccounts" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbtAdd" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtRemove" >
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
